nixos/doc/manual/release-notes/rl-2003.xml

```diff
@@ -85,7 +85,14 @@
 
   <itemizedlist>
    <listitem>
-    <para />
+    <para>
+      GnuPG is now built without support for a graphical passphrase entry
+      by default. Please enable the <literal>gpg-agent</literal> user service
+      via the NixOS option <literal>programs.gnupg.agent.enable</literal>.
+      Note that upstream recommends using <literal>gpg-agent</literal> and
+      will spawn a <literal>gpg-agent</literal> on the first invocation of
+      GnuPG anyway.
+    </para>
```

Comment:
Is udisks being re-enabled worth a note?

Reply:
It's already enabled for most desktop managers automatically. We just didn't enable it in the base installation device profile due to a big closure size.

Reply:
Correct, I meant as a change in the ISO's

Reply:
Only in the minimal iso - enabling plasma also enables udisks, and it defaults to true in the NixOS module system too.

Reply:
Still worth documenting that it's re-enabled in the minimal iso?

Reply:
It's not really a backwards-incompatible change (so it wouldn't belong in the GnuPG paragraph, and I also wouldn't consider the minimal iso now shipping udisk a "Highlight", so no, I wouldn't add something to the release notes.